STM32Cube/Core/Src/main.c

```diff
@@ -160,8 +163,9 @@ int main(void)
   MX_TIM1_Init();
   MX_ADC1_Init();
   MX_USART1_UART_Init();
+  MX_TIM2_Init();
   /* USER CODE BEGIN 2 */
-
+  HAL_TIM_Base_Start(&htim2);
```

Comment:
Just to confirm, this and the above config changes make the code generation happy? We don't get the "Freertos should use a timebase other than systick" warning?

Reply:
Well yes most of the config-related changes I suspect, are from this (I dont know for sure since I changed things in the IOC and the IDE auto-updates stuff, but yes, I did no other config changes). Specifically HTIM2 though is used for millis, HTIM6 is now used as the new time base for the system.